include/r3.h

```diff
@@ -46,7 +46,7 @@ typedef struct _node R3Node;
 typedef struct _R3Route R3Route;
 
 struct _node  {
-    R3Edge * edges;
```

Comment:
array of `R3Edge` gave memory leak and inconsistent memory administration.
I tried to close memory leaks with array of `R3Edge` and couldn't do it. Performance didn't decrease. 
So I use r3 in that way now.